src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -177,6 +142,46 @@ private void launchNode(SchedulerDriver driver, Offer offer,
     driver.launchTasks(Arrays.asList(offer.getId()), tasks);
   }
     
+  private ExecutorInfo createExecutor(String taskIdName, String nodeName, String executorName) {
+    int confServerPort = conf.getConfigServerPort();
+    List<Resource> resources = getExecutorResources();
+    ExecutorInfo executorInfo = ExecutorInfo.newBuilder()
+        .setName(nodeName + " executor")
+        .setExecutorId(ExecutorID.newBuilder().setValue("executor." + taskIdName).build())
+        .addAllResources(resources)
+        .setCommand(CommandInfo.newBuilder()
+        .addAllUris(Arrays.asList(
+            CommandInfo.URI.newBuilder().setValue(
+                conf.getExecUri())
+                .build(),
+            CommandInfo.URI.newBuilder().setValue(
+                String.format("http://%s:%d/hdfs-site.xml", localhost, confServerPort))
```

Comment:
Not something that needs to be fixed in this PR, but something that we should verify is that the use of IPv6 address work as the value of `localhost`.

Normally when you're using an IPv6 address in a URL the URL takes the form: `http://[1:0::1]:32100/file` notice the `[` and `]` surrounding the ip address so that the URL parser can correctly interpret ports.